src/clover/fileparser.py

```diff
@@ -2176,6 +2176,79 @@ def _parse_transmission_inputs(
         transmitters,
     )
 
+# Grid inputs file:
```

Comment:
Move these definitions up to the top of the file to match the existing ones

Reply:
?

Reply:
Fixed